src/com/estafet/learning/US005/Task5.java

```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
         factorialOfOddNumbers();
     }
 
-    public static int number;
+    private static int number;
```

Comment:
I mean it's a member of the class, dpo you need to share it's value among methods? If no - declare it as a local (method) variable.

Reply:
ok, changed.